### ISSUE TITLE
Handshake path 

### ISSUE THREAD

**Issue by naflan121** [vk496/linset#21]
Hi can i know where is the handshake file is stored, because when i attack an ap fake ap start with the previous handshaked ap's. How to fix this

**Comment by strasharo** [comment]
It's being saved in /tmp/TMPlinset/*.cap

**Comment by naflan121** [comment]
Hi i coulde'nt find the previous handshake file there and iam using wifislax

**Comment by strasharo** [comment]
In most of the modern distributions /tmp is tmpfs, which is recreated in the memory for every reboot.

**Comment by deltaxflux** [comment]
The handshake path included files, will get automaticly deleted after you close a session. Except you change LINSET_DEBUG=0 to LINSET_DEBUG=1

**Comment by naflan121** [comment]
i didnt change any option, i tried a attach but there are 3 fake api's including what i attack dont know how those api's are started and i tried to reinstall also same... Does alfa AWUS036H support for this attack?

**Comment by strasharo** [comment]
Yes, but only for airbase ap, hostapd is not supported with Realtek chipsets.
